<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNewTaskWizard</class>
 <widget class="QWizard" name="CNewTaskWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建任务</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>触发条件</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QGroupBox" name="groupBox_onceOrRepeat">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButton_once">
         <property name="text">
          <string>只执行一次</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_repeat">
         <property name="text">
          <string>重复</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QRadioButton" name="radioButton_timeTrigger">
         <property name="text">
          <string>时间</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_time">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QRadioButton" name="radioButton_countDown">
              <property name="text">
               <string>倒计时</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_timer">
              <property name="text">
               <string>定时</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QTimeEdit" name="timeEdit_time">
              <property name="minimumTime">
               <time>
                <hour>0</hour>
                <minute>1</minute>
                <second>0</second>
               </time>
              </property>
              <property name="displayFormat">
               <string>HH:mm</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_idleTrigger">
         <property name="text">
          <string>计算机空闲</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_idle">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QTimeEdit" name="timeEdit_idle">
              <property name="dateTime">
               <datetime>
                <hour>0</hour>
                <minute>0</minute>
                <second>1</second>
                <year>2000</year>
                <month>1</month>
                <day>1</day>
               </datetime>
              </property>
              <property name="minimumTime">
               <time>
                <hour>0</hour>
                <minute>0</minute>
                <second>1</second>
               </time>
              </property>
              <property name="displayFormat">
               <string>HH:mm:ss</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_netTrigger">
         <property name="text">
          <string>网络报文</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_netTrigger">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,1,0">
            <item>
             <widget class="QRadioButton" name="radioButton_http">
              <property name="text">
               <string>HTTP</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_udp">
              <property name="text">
               <string>UDP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="toolTip">
               <string>HTTP端口：20022
UDP端口：20023</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="qrc.qrc">:/image/help.png</pixmap>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLineEdit" name="lineEdit_ip">
              <property name="placeholderText">
               <string>对端IPv4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_netContent">
              <property name="placeholderText">
               <string>报文内容</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_fileTrigger">
         <property name="text">
          <string>发现文件时</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_fileTrigger">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="lineEdit_watchFile">
            <property name="placeholderText">
             <string>监视文件路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_explorer">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="qrc.qrc">
              <normaloff>:/image/folder.png</normaloff>:/image/folder.png</iconset>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_processTrigger">
         <property name="text">
          <string>进程运行/退出时</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_processTrigger">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QRadioButton" name="radioButton_processRunning">
              <property name="text">
               <string>进程运行</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_processNotRunning">
              <property name="text">
               <string>进程退出</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLineEdit" name="lineEdit_watchProcess">
              <property name="placeholderText">
               <string>监视进程名</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_selectProcess">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="qrc.qrc">
                <normaloff>:/image/search.png</normaloff>:/image/search.png</iconset>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="title">
    <string>状态条件</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_addCondition">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qrc.qrc">
          <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_removeCondition">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qrc.qrc">
          <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_or">
        <property name="text">
         <string>任一</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_and">
        <property name="text">
         <string>同时</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_condition"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage3">
   <property name="title">
    <string>执行内容</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_addAction">
        <property name="icon">
         <iconset resource="qrc.qrc">
          <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_removeAction">
        <property name="icon">
         <iconset resource="qrc.qrc">
          <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_up">
        <property name="icon">
         <iconset resource="qrc.qrc">
          <normaloff>:/image/up.png</normaloff>:/image/up.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_down">
        <property name="icon">
         <iconset resource="qrc.qrc">
          <normaloff>:/image/down.png</normaloff>:/image/down.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_action"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage4">
   <property name="title">
    <string>任务名称</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="lineEdit_taskName"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radioButton_timeTrigger</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_time</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_fileTrigger</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_fileTrigger</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_processTrigger</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_processTrigger</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_netTrigger</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_netTrigger</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_idleTrigger</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_idle</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
